langlib/langlib-test/src/test/resources/test-src/regexp_test.bal

```diff
@@ -1337,12 +1351,34 @@ function testReplaceAll() {
     var regExpr16_2 = re `AB|A|AK`;
     string result17_2 = regExpr16_2.replaceAll(str17, "Z");
     assertEquality("Z", result17_2);
+
+    var regExpr18 = re `\p{S}`;
+    string str18 = "🔜 #RayoBarça 🔵🔴 https://t.co/iHDUx7EmFJ";
+    string result18_1 = regExpr18.replaceAll(str18, "");
+    assertEquality(" #RayoBarça  https://t.co/iHDUx7EmFJ", result18_1);
+    string result18_2 = regExpr18.replaceAll(str18, "", 12);
+    assertEquality("🔜 #RayoBarça  https://t.co/iHDUx7EmFJ", result18_2);
```

Comment:
Shall we add a one example that replace with a unicode character.

```
    string str18 = "🔜 #RayoBarça 🔵🔴 https://t.co/iHDUx7EmFJ";
    string result18_1 = regExpr18.replaceAll(str18, "🔴");
```

Reply:
and something like

```
string str18 = "🔜 #RayoBarça 🔵🔴 https://t.co/iHDUx7EmFJ";
regexp:Span? result18_1 = re `🔵🔴`.find(str18);
```

Reply:
I tested in locally and both are working fine, better to add an test to cover this.